.github/workflows/formulaCI.yml

```diff
@@ -0,0 +1,86 @@
+name: Formula CI
+# Description: "Moving saphanabootstrap-formula from Travis to Github Actions"
+# The CI automatically publishes new releases every time a pull request is merged into master
+on: [push, pull_request]
+env:
+  PACKAGE_NAME: saphanabootstrap-formula 
+
+jobs:
+
+  validation:
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: Add salt repository
+      uses: myci-actions/add-deb-repo@4
+      with:
+        repo: deb http://repo.saltstack.com/apt/ubuntu/18.04/amd64/2019.2/ bionic main
+        key-server: http://repo.saltstack.com/apt/ubuntu/18.04/amd64/2019.2/SALTSTACK-GPG-KEY.pub
+    
+
+    - name: install salt
+      run: sudo apt-get install salt-common
+    - name: validation script
+      run: ci/validate-formula.sh
+
+
+  delivery:
+    needs: validation
+    runs-on: ubuntu-18.04
+    if: ${{ github.event_name != 'pull_request' }}  
+    container: 
+      image: shap/continuous_deliver
+      env:
+        OBS_USER: ${{ secrets.OBS_USER }}
+        OBS_PASS: ${{ secrets.OBS_PASS }}
+        OBS_PROJECT: ${{ secrets.OBS_PROJECT }}
+
+    steps:
+    - uses: actions/checkout@v2 
+      with:
+        fetch-depth: 0
+# OSC credentials must be configured beforehand as the HOME variables cannot be changed from /github/home
```

Comment:
Better to indent these comment to the correct level, to `- name` level.

Reply:
Hi @arbulu89 did you mean like this ? 
-name: 
# comment 
# comment